client/src/main/com/sinch/sdk/domains/verification/adapters/VerificationService.java

```diff
@@ -22,28 +24,33 @@ public class VerificationService implements com.sinch.sdk.domains.verification.V
   private final HttpClient httpClient;
   private VerificationsService verifications;
   private StatusService status;
-  private final Map<String, AuthManager> authManagers;
+  private Map<String, AuthManager> authManagers;
 
   public VerificationService(Configuration configuration, HttpClient httpClient) {
     this.configuration = configuration;
     this.httpClient = httpClient;
 
+    // by default, unified id/secret from configuration are used but can be super sed if
+    // "useSecrets"
+    // is called after initialization
+    useSecrets(
```

Comment:
- replaced by `setApplicationCredentials` to stay with the setter pattern used by SDK
- removed the default configuration usage based onto unified credentials because of not yet available (to avoid misunderstanding about what is supported and inputs)
- added a check against undefined credentials before calling underlying service to have exception message from SDK and not from server